### PR TITLE
add style to variabless.scss and add mixins.scss

### PR DESCRIPTION
**Pull request by Mariusz-web**

### CODE REVIEW

File: src/styles/variables.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
+$font-style: Roboto;
+
+$small-font: 14px;
```
Comment (Veryyapeee) [inline src/styles/variables.scss:3]: sizes for fonts and icons should be in EM/REM

File: src/styles/variables.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+$image-width-large: 180px;
+
+$primary-color: #FFFFFF;
+$secondary-color: #776E6E;
+$accent-color: #FCC67C;
+$btn-color: #000000;
+
+$primary-backgroundColor: #000000;
+$secondary-backgroundColor: #171717;
+$comment-backgroundColor: rgba(37, 37, 37, 0.5);
```
Comment (Veryyapeee) [inline src/styles/variables.scss:19]: change it for hex (for example $comment-backgroundColor would be: `#25252580`)

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

File: src/styles/variables.scss (hunk 1 above)
Comment (Mariusz-web) [reply src/styles/variables.scss]: done

File: src/styles/variables.scss (hunk 2 above)
Comment (Mariusz-web) [reply src/styles/variables.scss]: done

**Review by Veryyapeee — APPROVED** [review]

**Review by KonTra7cf — APPROVED** [review]

**Review by wikbro7650 — APPROVED** [review]

**Pull request merged**